internal/debug/flags.go

```diff
@@ -55,6 +55,7 @@ var (
 	memprofilerateFlag = cli.IntFlag{
 		Name:  "memprofilerate",
 		Usage: "Turn on memory profiling with the given rate",
+		Value: runtime.MemProfileRate,
```

Comment:
It basically restores the default value. Memory profiling is always "on" in most programs, but has low impact on performance.